### PR TITLE
Add support for organizations

### LINKED ISSUE #4: Handle organizations
Currently we support aggregating stars of every repo under a given user (like `star-history dtolnay`) but not every repo under an organization (like `star-history async-rs`). Currently that fails with:

```console
$ star-history async-rs
Error from GitHub api: Could not resolve to a User with the login of 'async-rs'.
```

We should treat arguments that are organizations much like users, adding up the stars across all their repos.

### PR DESCRIPTION
**Pull request by Gingeh**
Fixes #4.

### Solution
Use `query.repositoryOwner` instead of `query.user` in `query_user`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,16 +25,16 @@ $ cargo install star-history
 ## Usage
 
 We require a token for accessing GitHub's GraphQL API. Head to
-https://github.com/settings/tokens and click "Generate new token". The default
-public access permission is sufficient &mdash; you can leave all the checkboxes
-empty. Save the generated token somewhere like ~/.githubtoken.
+https://github.com/settings/tokens and click "Generate new token". Organizations require
+the `read:org` permission. Save the generated token somewhere like ~/.githubtoken.
```
Comment (dtolnay) [inline README.md]: I tested this PR with a token that has only "public access" permission (no `read:org`) and it seems to work. Is the permission only relevant in the case that you want to count stars of a private repo?

**Review by dtolnay — COMMENTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (Gingeh) [reply README.md]: I got an error while trying `query.organization`, but I didn't test whether it was still required for `query.repositoryOwner`. 
It works without `read:org` now, thanks for pointing that out!

**Review by dtolnay — APPROVED** [review]

**Comment by dtolnay** [comment]
Published in star-history 1.0.4.

**Pull request merged**